web/views.py

```
@@ -845,6 +847,71 @@
+        submission_count = 0
+        if is_teacher:
+            submission_count = sum(1 for q in exam.user_quizzes.all() if q.completed)
+
+        course_exam_data.append(
+            {
+                "exam": exam,
+                "user_attempt": user_attempt,
+                "submission_count": submission_count,
+                "user_attempts_count": user_attempts_count,
+                "user_remaining_attempts": exam.max_attempts - user_attempts_count,
+            }
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)